src/components/RegistrationForm.tsx

```diff
@@ -353,7 +353,8 @@ export default function RegistrationForm({ handles }) {
   const usernameValidator = useCallback(
     (value: string) => {
       const validationErrors: (string | React.ReactNode)[] = [];
-      if (handles.has(value)) {
+      const lowerCaseHandles = handles.map((handle) => handle.toLowerCase());
+      if (lowerCaseHandles.has(value.toLowerCase)) {
```

Comment:
```suggestion
      if (lowerCaseHandles.has(value.toLowerCase())) {
```

Reply:
This also might be better as handles.find() instead of constructing an entire list of lowercase handles to then search through.

Reply:
Good idea

Reply:
@joshuashort can you take another look?